File: 안남규/2019-01-01_code.py
def solution(s):
    tmp_1 = list(s)
    tmp_2 = [0]

    for i in tmp_1:
        if tmp_2[-1] == i:
            tmp_2.pop()
            continue

        tmp_2.append(i)

    tmp_2.remove(0)

    if not tmp_2:
        return 1

    return 0

if __name__ == '__main__':
    print(solution('baabaa'))

File: 권하경/2019_01_01/2019_01_01_code.js
function solution(s){
    let answer=0;
    let Arr = [];
    let j=0;
    if(s.length%2!=0) return answer;

    for(let i=0;i<s.length;i++){
        Arr.push(s[i]);

        if (Arr[j]==Arr[j-1]){
            Arr.pop();
            Arr.pop();
            j=Arr.length-1;
        }
        j++;
    }
    if(Arr.length==0) answer=1;
    return answer;
}

var s = 'baabaa';
var result = solution(s);
console.log(result); 